<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateDialog</class>
 <widget class="QDialog" name="UpdateDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>872</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>学号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>姓名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>72</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>性别：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>72</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>年级：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>280</y>
     <width>72</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>院系：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>330</y>
     <width>72</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>专业：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>380</y>
     <width>72</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>班级：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>430</y>
     <width>72</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>邮箱：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>480</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>联系方式：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="studentIdLineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>如：201642300001</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameLineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>如：张三</string>
   </property>
  </widget>
  <widget class="QComboBox" name="sexComboBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>男</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>女</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="majorLineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>320</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>计算机科学与技术</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="emailLineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>420</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>如：12345678@address.com</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="telephoneLineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>470</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>如：13200001111</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fileOpenButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>330</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>（可选）修改照片</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>560</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>560</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>530</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>密码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordLineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>520</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>默认为空不修改</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="rePasswordLineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>570</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>默认为空不修改</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>580</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>再输一次：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="departmentLineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>270</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>如：信息工程学院</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="gradeDateEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>170</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2016</year>
     <month>1</month>
     <day>1</day>
    </datetime>
   </property>
   <property name="displayFormat">
    <string>yyyy</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="classesSpinBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>370</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="photoView">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>20</y>
     <width>384</width>
     <height>288</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="labComboBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>220</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>72</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>实验室：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cameraOpenButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>380</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>（可选）拍照</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>studentIdLineEdit</tabstop>
  <tabstop>nameLineEdit</tabstop>
  <tabstop>sexComboBox</tabstop>
  <tabstop>gradeDateEdit</tabstop>
  <tabstop>labComboBox</tabstop>
  <tabstop>departmentLineEdit</tabstop>
  <tabstop>majorLineEdit</tabstop>
  <tabstop>classesSpinBox</tabstop>
  <tabstop>emailLineEdit</tabstop>
  <tabstop>telephoneLineEdit</tabstop>
  <tabstop>passwordLineEdit</tabstop>
  <tabstop>rePasswordLineEdit</tabstop>
  <tabstop>fileOpenButton</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
